pkg/server/server.go

```diff
@@ -185,7 +185,7 @@ func WithResolveNodeLimit(limit uint32) OpenFGAServiceV1Option {
 }
 
 // WithResolveNodeBreadthLimit sets a limit on the number of goroutines that can be created
-// when evaluating a subtree of a Check or ListObjects call.
+// when evaluating a subtree of a Check, ListObjects or ListUsers call.
```

Comment:
nit (non-blocking): Same suggestions as above. Let's maybe consider making this more generic for long term adaptation.

> ... when evaluating a subtree of an FGA query.